constellations/gemini-admin/src/achieve/achieve.sass

```diff
@@ -47,4 +47,4 @@ a.add-achievement
     background-color: $colorTurquoise
     color: $colorWhite
     padding: 0.75em
-    border-radius: 10px
+    border-radius: 10px
```

Comment:
everything looks fine, a couple of css ordering things though: padding & border before colors